spec.bs

```diff
@@ -178,6 +178,9 @@ spec: shared-storage-explainer; urlPrefix: https://github.com/WICG/shared-storag
 spec: UUID; urlPrefix: https://www.ietf.org/rfc/rfc4122.txt
     type: dfn
         text: urn uuid; url: urn-uuid
+spec: hr-time; urlPrefix: https://w3c.github.io/hr-time/
```

Comment:
I just added this anchor because suddenly the spec stopped being able to find [=/current wall time=] on its own.